lib/galaxy/jobs/runners/kubernetes.py

```diff
@@ -18,18 +18,26 @@
 )
 from galaxy.jobs.runners.util.pykube_util import (
     DEFAULT_JOB_API_VERSION,
+    delete_ingress,
+    delete_job,
+    delete_service,
     ensure_pykube,
+    find_ingress_object_by_name,
     find_job_object_by_name,
     find_pod_object_by_name,
+    find_service_object_by_name,
     galaxy_instance_id,
     is_pod_unschedulable,
+    Ingress,
+    ingress_object_dict,
```

Comment:
```suggestion
    Ingress,
    ingress_object_dict,
    is_pod_unschedulable,
```
To make the linter happy.